src/clj/rems/css/styles.clj

```diff
@@ -347,6 +347,56 @@
    [:.collapsing {:-webkit-transition "height 0.25s linear"
                   :-o-transition "height 0.25s linear"
                   :transition "height 0.25s linear"}]]
+
+  ;; autocomplete, duplicates some Bootstrap styling
+  ;; because the component classes are hard-coded
+  [:.autocomplete {:width (u/percent 100)}
+   [:.autocomplete__control
+    [:input {;;:padding (u/px 10)
+             ;;:border-radius (u/px 4)
+             ;;:border [[(u/px 1) :solid "#111"]]
+
+             ;; from Bootstrap .form-control
+             :display :block
+             :width (u/percent 100)
+             :padding [[(u/rem 0.375) (u/rem 0.75)]]
+             :font-size (u/rem 1)
+             :line-height 1.5
+             :color "#495057"
```

Comment:
These random hardcoded colours should conform to existing theme and probably be configurable as well